test/dict.jl

```diff
@@ -323,6 +325,13 @@ end
         @test !isempty(summary(keys(d)))
         @test !isempty(summary(values(d)))
     end
+    # show on empty Dict
+    io = IOBuffer()
+    d = Dict{Int, String}()
+    show(io, d)
+    str = String(take!(io))
+    @test str == "Dict{Int64,String}()"
```

Comment:
This needs to be `Dict{Int,String}()`.

Reply:
That doesn’t seem right. 

This is testing the Dict show function, which should show `Dict{Int64,String}()` on 64-but machines, and `Dict{Int32,String}()` on 32-but machines.

To make it more consistent, it would be better to be explicit about the integer type when constructing the Dict.

Reply:
`"Dict{$(Int),String}()"`